### ISSUE TITLE
[JS|.NET HTML] [Accessibility] tabindex attribute should be added to items with a selectAction

### ISSUE THREAD

**Issue by matthidinger** [microsoft/AdaptiveCards#2326]
# Platform

What platform is your issue or question related to? (Delete other platforms).

* .NET HTML
* JavaScript

# Author or host

Host

# Version of SDK

all

# Issue

We are not properly adding a HTML `tabindex` attribute when a `Container`, `Image`, `ColumnSet`, or `Column` has a `selectAction`. NOTE: This should only be added when `SupportsInteractivity` is true

**Comment by shalinijoshi19** [comment]
@dclaux is this still an issue today?

**Comment by jonmill** [comment]
This should be fixed
